chatAppTemplate_2/chatApp.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
```

Comment:
Can you update indentation here? Either 2 spaces or 4... not 2, 4, 8, or random.